app/controllers/articles_controller.rb

```diff
@@ -3,22 +3,27 @@ class ArticlesController < ApplicationController
   before_action :set_article, only: [:show, :edit, :update, :destroy]
   after_action :verify_authorized, except: :index
 
+
   def index
     @articles = Article.all
     authorize @articles
+    set_meta_tags title: 'Articles'
   end
 
   def show
+    set_meta_tags @article
   end
 
   # GET /articles/new
   def new
     @article = current_user.articles.new
     authorize @article
+    set_meta_tags title: 'New Article'
   end
 
   # GET /articles/1/edit
   def edit
+    set_meta_tags title: 'Edit ' + @article.title
```

Comment:
use "Edit #{@article.title}" which is a better ruby style instead of adding strings like this